platform_channels/lib/src/method_channel_demo.dart

```diff
@@ -84,7 +84,7 @@ class _MethodChannelDemoState extends State<MethodChannelDemo> {
   }
 
   void showErrorMessage(BuildContext context, String errorMessage) {
-    Scaffold.of(context).showSnackBar(SnackBar(
+    ScaffoldMessenger.of(context).showSnackBar(SnackBar(
```

Comment:
Since, we are now using `ScaffoldMessenger`, we can also remove the Builder on line 25.